reciprocalspaceship/stats/completeness.py

```diff
@@ -3,7 +3,7 @@
 
 import reciprocalspaceship as rs
 from reciprocalspaceship.utils.asu import in_asu
-from reciprocalspaceship.utils.binning import assign_with_binedges, bin_by_percentile
+from reciprocalspaceship.utils.binning import assign_with_binedges
```

Comment:
I think you can accomplish the same thing you are using `assign_with_binedges` to do by using `groupby(['H', 'K', 'L']).first()`. I'm not sure if that would be better or not. Just thinking out loud.

Reply:
I don't quite follow -- the `groupby()` call would return the first instances of each Miller index, but `assign_with_binedges` returns which resolution bin each reflection gets assigned to. In this case, the goal is to use the resolution bins determined for the provided `dataset` to assign the reflections in the 100% complete instance. 

If we make your above suggestion (to have `assign_resolution_bins()`) take bin edges, then we could refactor this to use that function instead of `utils`.

Reply:
okay we should probably do it via an `assign_resolution_bins` which accepts bin edges.